aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -109,16 +109,18 @@ void check_all_any(const char* name, const Tensor& self, const Tensor& result) {
   }
 }
 
-TORCH_META_FUNC2(all, dim)(const Tensor& self, int64_t dim, bool keepdim) {
+TORCH_PRECOMPUTE_META_FUNC2(all, dim)(const Tensor& self, int64_t dim, bool keepdim) {
   check_all_any("all", self, maybe_get_output());
   auto out_dtype = get_result_or_bytebool_dtype(self, maybe_get_output());
   resize_reduction(*this, self, dim, keepdim, out_dtype);
```

Comment:
This was added while I was working on this PR and calls `maybe_wrap_dim` internally, so we're still calling that twice (in the meta now instead of once in the meta and once in the impl). Not sure what to do here; having `resize_reduction` return the result doesn't make sense.

Reply:
bloop